parallel/parallel_collections.py

```diff
@@ -30,9 +35,9 @@ def __init__(self, predicate):
         
     def __call__(self, item):
         if self.predicate(item):
-            return (True, item)
+            return EvalResult(value=True, arg=item)
         else:
-            return (False, None)
+            return EvalResult(value=False, arg=None)
 
         
```

Comment:
How do you feel about using 'result' instead of 'value' and 'item' instead of 'arg'. Since it's called 'item' in the rest of the _Filter, and value makes it sound like it's the value that was passed to the predicate. Perhaps bool is clearer. What do you think?